### PR TITLE
add arguments for number of MPI process

### PR DESCRIPTION
**Pull request by jmkerloch**
- Move arguments parsing from config.py to driver.py.
- Add number of MPI process argument

### CODE REVIEW

File: src/src_python/antares_xpansion/driver.py [hunk 1]
```diff
@@ -29,7 +30,38 @@ def __init__(self, config):
...
+                            required=True)
+        parser.add_argument("--method",
+                            dest="method",
+                            type=str,
+                            choices=["mpibenders", "mergeMPS", "both", "sequential"],
+                            help="Choose the optimization method")
+        parser.add_argument("-c",
+                            dest="c",
+                            help='Name of the file to use for exclusion constraints')
+        parser.add_argument("-n", "-np",
```
Comment (sgatto) [inline src/src_python/antares_xpansion/driver.py:59]: je ne suis pas sur que `-np` soit accepté par add_argument, normalement un `--` est nécessaire

**Review by sgatto — COMMENTED** [review]
I'm not convinced to give the responsibility of parsing the argument to `driver` instad of `xpansionConfig.py`. I'd rather give the responsibility to xpansionConfig instead of driver.
I can propose the changes

File: src/src_python/antares_xpansion/driver.py (hunk 1 above)
Comment (jmkerloch) [reply src/src_python/antares_xpansion/driver.py]: En effet. Remarques prises en compte.

**Comment by jmkerloch** [comment]
> 
> 
> I'm not convinced to give the responsibility of parsing the argument to `driver` instad of `xpansionConfig.py`. I'd rather give the responsibility to xpansionConfig instead of driver.
> I can propose the changes

I agree, changes done. `xPansionConfig `now store the configuration parsed.

Added also a check on N_MPI parameter.

**Review by sgatto — APPROVED** [review]
OK

**Pull request merged**